web/src/pages/Auth.tsx

```diff
@@ -11,7 +11,7 @@ import "../less/auth.less";
 
 const validateConfig: ValidatorConfig = {
   minLength: 4,
-  maxLength: 24,
+  maxLength: 100,
```

Comment:
How about `320`?

<img width="672" alt="image" src="https://user-images.githubusercontent.com/24653555/203168568-5afe7a4e-4bf9-4214-99c1-34fe5f169796.png">

Reply:
Yes, that makes sense